frontend/src/tests/common/fixPrimevueCssErr.ts

```diff
@@ -0,0 +1,12 @@
+//fix "Could not parse CSS stylesheet" with the primevue styling
+//https://github.com/primefaces/primevue/issues/4512
+//https://stackoverflow.com/questions/69906136/console-error-error-could-not-parse-css-stylesheet/69958999#69958999
+export const fixPrimevueCssError = () => {
+const originalConsoleError = console.error;
+const jsDomCssError = 'Error: Could not parse CSS stylesheet';
+console.error = (...params) => {
+    if (!params.find((p) => p.toString().includes(jsDomCssError))) {
+        originalConsoleError(...params);
+    }
+};
+}
```

Comment:
After Primevue switched to CSS Layers the tests are having problems parsing the stylesheet. This deals with the errors on the console. I'm using this fix since a did not find a better one
https://github.com/primefaces/primevue/issues/4512
https://stackoverflow.com/questions/69906136/console-error-error-could-not-parse-css-stylesheet/69958999#69958999

Reply:
yeah, just put a link here as our notes... I didn't find a better solution for now, most of the examples use this way. We can chat more tomorrow. thanks!! https://github.com/primefaces/primeng/issues/14085#issuecomment-1888629612

Reply:
let's rename this file to be `fixJsdomCssErr`, it's actually the jsdom package is causing the error

Reply:
Sure, I'll rename it!